app/javascript/packs/application/redux/tours/actions.js

```diff
@@ -72,3 +75,16 @@ export const getTour = tourId => async dispatch => {
     dispatch({ type: GET_TOUR_ERROR });
   }
 };
+
+export const deleteTour = tourId => async dispatch => {
+  dispatch({ type: DELETE_TOUR });
+  const response = await api.deleteTour(tourId);
+
+  if (response.ok) {
+    dispatch({ type: DELETE_TOUR_SUCCESS });
```

Comment:
it seems like your reducer is expecting you to dispatch this action with data...?

However, I don't think deleting a tour should provide data. Instead, I think this dispatch should look like this:

```javascript
dispatch({ type: DELETE_TOUR_SUCCESS, tourId });
```

Then, the reducer should remove that tour from state.